read-skipfile-results

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: set ts=4
+#
+# Copyright 2023-present Linaro Limited
+#
+# SPDX-License-Identifier: MIT
+
+from argparse import ArgumentParser
+from copy import deepcopy
+from datetime import datetime
+from json import load
+from logging import DEBUG, INFO, basicConfig, getLogger
+from os import chdir, environ, getenv, path, popen, rename
+from pathlib import Path
+from re import findall, sub
+
+import pandas as pd
+from git import Repo
+from github import Github
+from ruamel.yaml import YAML
+from squad_client.core.api import SquadApi
+from squad_client.core.models import Environment, Squad, TestRun
+from squad_client.utils import first, getid
+
+from squadutilslib import get_file
+
+
+def parse_args(raw_args):
+    parser = ArgumentParser(description="Read results and update skipfile")
+
+    parser.add_argument(
+        "--group-name",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--run-count",
+        required=True,
+        type=int,
+        help="The number of runs performed.",
+    )
+
+    parser.add_argument(
+        "--builds-file",
+        required=False,
+        default="builds_for_skipfile_runs.txt",
+        help="File containing the list of SQUAD build names",
+    )
+
+    parser.add_argument(
+        "--debug",
+        required=False,
+        action="store_true",
+        default=False,
+        help="Display debug messages.",
+    )
+
+    parser.add_argument(
+        "--github-token",
+        required=False,
+        default="GITHUB_ACCESS_TOKEN",
+        help="The name of the environment variable containing the Github API token.",
+    )
+
+    parser.add_argument(
+        "--github-push",
+        required=False,
+        default=True,
+        type=int,
```

Comment:
should not be int.
Default should probably be False so you don't push... and if you push you just add --github-push or --push-to-github or ...